Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/Definitions/Input.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.Salesforce.CreateSObject.Definitions;
+/// <summary>
+/// Input-class for CreateSObject-task.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Salesforce Domain.
+    /// /services/data/v52.0/query will be added automatically, so the domain is enough.
+    /// </summary>
+    /// <example>https://example.my.salesforce.com</example>
+    [DefaultValue(@"https://example.my.salesforce.com")]
+    public string Domain { get; set; }
```

Comment:
Actually you need to set the display format to Text, since default is expression. This was also my mistake, sorry for the trouble.